spec/lib/salus/processor_spec.rb

```diff
@@ -1,17 +1,19 @@
 require_relative '../../spec_helper.rb'
 
 RSpec::Matchers.define :match_report_json do |expected|
-  def remove_key(json_string, key = 'running_time')
+  def remove_keys(json_string)
```

Comment:
Could you revert this change?  Removing the extra parameter may break some of our private tests.